ntc_templates/templates/eltex_show_ip_interface.textfsm

```diff
@@ -0,0 +1,53 @@
+Value GATEWAY_IP (\S+)
+Value GATEWAY_STATUS (\S+)
+Value GATEWAY_TYPE (\S+)
+Value IP (\S+)
+Value I_F (.*?)
```

Comment:
```suggestion
Value I_F (.*?)
```

Is that an interface? Let's update this.

Reply:
Yes. This is interface. I'll fix it